src/avrouting.cs

```diff
@@ -0,0 +1,94 @@
+//
+// AVRouting bindings
+//
+// Authors:
+//	TJ Lambert  <TJ.Lambert@microsoft.com>
+//
+// Copyright 2022 Microsoft Corp. All rights reserved.
+//
+
+using System;
+using ObjCRuntime;
+using Foundation;
+using UniformTypeIdentifiers;
+using AVKit;
+using OS_nw_endpoint = System.IntPtr;
```

Comment:
Hmm interesting, @rolfbjarne  should this be a NativeHandle in NET?

Reply:
Yes, a `NativeHandle` would be better, although this isn't visible to the user (if your second comment is fixed too).